File: baekjoon/java/25314. 코딩은 체육과목 입니다.java
import java.util.Scanner;

public class Main{

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int nbyte = sc.nextInt();
        String btype = "long";

        for (int i=0;i<nbyte/4;i++){
            System.out.print(btype+" ");
        }
        System.out.println("int");
    }
}

File: baekjoon/cpp/25314. 코딩은 체육과목 입니다.cpp
#include <string>
#include <iostream>
using namespace std;


int main(void){

    int nbyte;
    string btype = "long";
    
    cin >> nbyte;
    for (int i=0;i<nbyte/4;i++){
        cout << btype << " ";
    }
    cout << "int";
    return 0;
}
